addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -735,7 +735,7 @@ const Wysiwyg = Widget.extend({
         if ($wrapwrap.length) {
             $('#wrapwrap')[0].removeEventListener('scroll', this.odooEditor.multiselectionRefresh, { passive: true });
         }
-        $(this.$root).off('mousedown');
+        $(this.$root).off('click');
```

Comment:
This is changed by the defition of the listener is not changed. Either this is unintended, or the code currently never stop those listeners.

Reply:
See https://github.com/odoo/odoo/pull/108087/files#diff-aa4686cd9f6a0d50e1040ba3dda0b29e9898e7cc31a47f1f02b46a2c4c047a72L342
I noticed that someone had changed the definition of the listener without adapting this line so indeed the code currently never stops that listener and that is fixed with this commit as well.

By the way I could also remove that `$(...)` around it, I don't know why it's there since it's already a jquery object as indicated by its name.